docs/versioning/how-to-version.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+nav_order: 4
+title: How to create versions  
+has_children: false
+parent: Versioning
+---
+# How to create versions in the Registry
+
+## Step 1. Navigate to the item record in the Registry
+
+- Ensure you're logged in.
+- Navigate to the item record in the Registry
+- Click on the "VERSIONING" tab as shown below
```

Comment:
The VERSIONING tab may not exist - need to add an info/warning block here to explain why it might not be there.

Reply:
added pre-requisites